[IMP] empty

arpi-odoo opened the pull request.

arpi-odoo committed:
[FIX] hr: new contract button invisible

arpi-odoo committed:
[IMP] hr_skills: cleanup certification reporting

arpi-odoo committed:
[FIX] hr_holidays: fix traceback leave type

arpi-odoo committed:
[FIX] hr: fix delete version

arpi-odoo committed:
[FIX] hr: fix various bugs

mepe-odoo committed:
[FIX] hr_holidays: fix accrual day selection

arpi-odoo committed:
[IMP] empty

robodoo commented:
This PR targets the un-managed branch odoo-dev/odoo:master-hr-onboarding, it needs to be retargeted before it can be merged.

abdullahsherif committed:
[IMP] hr_holidays: making day selection field depending on the month

Before, you could select day 31 for 30/29/28 days month in accural plan levels

added custom widget for day selection to adapt with the selected month

Task #5404372
